pkg/tsdb/elasticsearch/response_parser_frontend_test.go

```diff
@@ -0,0 +1,1686 @@
+						  {
+							"p_1": { "values": { "75": 3.3, "90": 5.5 } },
+							"doc_count": 10,
+							"key": 1000
+						  },
+						  {
+							"p_1": { "values": { "75": 2.3, "90": 4.5 } },
+							"doc_count": 15,
+							"key": 2000
+						  }
+						]
+					  }
+					}
+				  },
+				  {
+					"aggregations": {
+					  "e_3": {
+						"buckets": [
+						  {
+							"key": "server1",
+							"e_4": {
+							  "buckets": [
+								{
+								  "e_1": {
+									"max": 10.2,
+									"min": 5.5,
+									"std_deviation_bounds": { "upper": 3, "lower": -2 }
+								  },
+								  "doc_count": 10,
+								  "key": 1000
+								}
+							  ]
+							}
+						  },
+						  {
+							"key": "server2",
+							"e_4": {
+							  "buckets": [
+								{
+								  "e_1": {
+									"max": 10.2,
+									"min": 5.5,
+									"std_deviation_bounds": { "upper": 3, "lower": -2 }
+								  },
+								  "doc_count": 10,
+								  "key": 1000
+								}
+							  ]
+							}
+						  }
+						]
+					  }
+					}
+				  },
+				  {
+					"hits": {
+					  "total": {
+						"relation": "eq",
+						"value": 1
+					  },
+					  "hits": [
+						{
+						  "_id": "6",
+						  "_type": "_doc",
+						  "_index": "index",
+						  "_source": { "sourceProp": "asd" }
+						}
+					  ]
+					}
+				  }
+				]
+			  }
+			`)
+
+	result, err := queryDataTest(query, response)
+	require.NoError(t, err)
+
+	verifyFrames := func(name string, expectedLength int) {
+		r, found := result.response.Responses[name]
+		require.True(t, found, "not found: "+name)
+		require.NoError(t, r.Error)
+		require.Len(t, r.Frames, expectedLength, "length wrong for "+name)
+	}
+
+	verifyFrames("COUNT_GROUPBY_DATE_HISTOGRAM", 1)
+	verifyFrames("COUNT_GROUPBY_HISTOGRAM", 1)
+	// verifyFrames("RAW_DOC", 1) // FIXME
+	verifyFrames("PERCENTILE", 2)
+	verifyFrames("EXTENDEDSTATS", 4)
+	// verifyFrames("D", 1) // FIXME
+}
+
+func TestSimpleQueryReturns1Frame(t *testing.T) {
+	query := []byte(`
+		[
+			{
+				"refId": "A",
+				"timeField": "t",
+				"metrics": [{ "type": "count", "id": "1" }],
+				"bucketAggs": [
+				{ "type": "date_histogram", "field": "@timestamp", "id": "2" }
+				]
+			}
+			]
+		`)
+
+	response := []byte(`
+		{
+			"responses": [
+			  {
+				"aggregations": {
+				  "2": {
+					"buckets": [
+					  { "doc_count": 10, "key": 1000 },
+					  { "doc_count": 15, "key": 2000 }
+					]
+				  }
+				}
+			  }
+			]
+		  }
+		`)
+
+	result, err := queryDataTest(query, response)
+	require.NoError(t, err)
+
+	require.Len(t, result.response.Responses, 1)
+	frames := result.response.Responses["A"].Frames
+	require.Len(t, frames, 1, "frame-count wrong")
+	frame := frames[0]
+	// require.Equal(t, "Count", frame.Name) // FIXME
+
+	requireFrameLength(t, frame, 2)
+	requireTimeValue(t, 1000, frame, 0)
+	requireNumberValue(t, 10, frame, 0)
+}
+
+func TestSimpleQueryCountAndAvg(t *testing.T) {
+	query := []byte(`
+	[
+		{
+			"refId": "A",
+			"timeField": "t",
```

Comment:
I'd personally use `"timeField": "@timestamp",` everywhere as we use in all other tests, but that is just nit pick.

Reply:
i see your point, but i'd argue we should use some other value than `@timestamp`, the reason is, we default to `@timestamp` when there is no time-field in the datasource-config, so if we want to make sure a customized time-field-value works, we should find a non-default value. anyway, can talk talk about later 👍 , i can change it, it's not a big deal.

(or we can just drop this field completely.. probably).